### ISSUE TITLE
Installing “font-viewer@0.4.0” failed

### ISSUE THREAD

**Issue by matteobertoldo** [ericfreese/font-viewer#24]
Installing “font-viewer@0.4.0” failed.

> freetype2@0.3.0 install /private/var/folders/1x/b3swxcss1plfmjf2cm06r0jc0000gn/T/apm-install-dir-202095-4193-bpfitf.dsclj/node_modules/font-viewer/node_modules/freetype2
> node-gyp rebuild

No receipt for 'com.apple.pkg.CLTools_Executables' found at '/'.

No receipt for 'com.apple.pkg.DeveloperToolsCLILeo' found at '/'.

No receipt for 'com.apple.pkg.DeveloperToolsCLI' found at '/'.

gyp: No Xcode or CLT version detected!
gyp ERR! configure error 
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/Applications/Atom.app/Contents/Resources/app/apm/node_modules/npm/node_modules/node-gyp/lib/configure.js:351:16)
gyp ERR! stack     at ChildProcess.emit (events.js:198:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:248:12)
gyp ERR! System Darwin 19.6.0
gyp ERR! command "/Applications/Atom.app/Contents/Resources/app/apm/bin/node" "/Applications/Atom.app/Contents/Resources/app/apm/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /private/var/folders/1x/b3swxcss1plfmjf2cm06r0jc0000gn/T/apm-install-dir-202095-4193-bpfitf.dsclj/node_modules/font-viewer/node_modules/freetype2
gyp ERR! node -v v10.20.1
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok 
npm WARN enoent ENOENT: no such file or directory, open '/private/var/folders/1x/b3swxcss1plfmjf2cm06r0jc0000gn/T/apm-install-dir-202095-4193-bpfitf.dsclj/package.json'
npm WARN apm-install-dir-202095-4193-bpfitf.dsclj No description
npm WARN apm-install-dir-202095-4193-bpfitf.dsclj No repository field.
npm WARN apm-install-dir-202095-4193-bpfitf.dsclj No README data
npm WARN apm-install-dir-202095-4193-bpfitf.dsclj No license field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! freetype2@0.3.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the freetype2@0.3.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/walit/.atom/.apm/_logs/2020-10-05T13_41_55_817Z-debug.log

**Comment by aminya** [comment]
Which version of Atom are you using? This log shows Node 10 which is quite old.

**Comment by matteobertoldo** [comment]
Hi mate! @aminya I still removed Xcode from my Mac a few months ago because it was taking up too much space. 
And removing it would seem like there are problems with `gyp` see [this](https://medium.com/flawless-app-stories/gyp-no-xcode-or-clt-version-detected-macos-catalina-anansewaa-38b536389e8d).

Seems to be tied to various other `node` packages that require it, in fact` freetype2` calls `gyp`. I currently don't have any modules installed that display fonts.

Now I try to follow the steps on the page I linked here and I try to reinstall the package. I currently use Atom 1.51.0.

**Comment by aminya** [comment]
We can potentially prebuild the binaries using node prebuild. I may look into doing that if I find some free time.

https://github.com/prebuild/node-gyp-build

**Comment by matteobertoldo** [comment]
@aminya Thanks for the support. I'll give you an update regarding the installation of the package. I followed the steps of the page in Medium that I had indicated in the previous comment...but I installed the Command Line Tools from the official Apple [website](https://developer.apple.com/download/more/) for Developers and now it works correctly!

Here a visual example after installing the package...awesome!

<img width="1392" alt="ortica" src="https://user-images.githubusercontent.com/15775323/95342035-44214300-08b7-11eb-8b6d-70eec7129ab8.png">


P.S. It does not seem to support the `woff2` format though. For now, thank you very much.
